src/style.css

```diff
@@ -1 +1,133 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+:root {
+  --font-family: Avenir, sans-serif;
+  --font-size-base: 16px;
+  --font-size-small: 14px;
+  --color-text: #3d4e61;
+  --color-border: #e1e7ed;
+  --box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  --box-shadow-hover: 0 3px 8px rgba(61, 78, 97, 0.2);
+  --gradient-focus: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+html {
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's recommended to style elements using classes instead of tags to avoid conflicts and increase reusability. However, setting `box-sizing: border-box;` on the `html` element is a common practice to ensure consistent box-sizing across the entire document.